lib/meta_tags.rb

```diff
@@ -1,5 +1,6 @@
 require 'action_controller'
 require 'action_view'
+require 'rails-html-sanitizer'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.